services/src/lib.rs

```diff
@@ -50,3 +50,8 @@ pub mod workflows;
 pub mod pro;
 
 pub use geoengine_datatypes::test_data;
+
+// re-export test macro
```

Comment:
why is it necessary to do thsi?

Reply:
You can get nicer names and also put it into sub-modules